src/routes/UserRecordContainer.js

```diff
@@ -56,6 +57,11 @@ class UserRecordContainer extends React.Component {
       path: 'addresstypes?query=cql.allRecords=1 sortby desc',
       records: 'addressTypes',
     },
+    departments: {
+      type: 'okapi',
+      path: 'departments?query=cql.allRecords=1',
+      records: 'departments',
+    },
```

Comment:
FYI, even with `cql.allRecords=1`, this query will only retrieve 10 records unless you add a `limit=N` clause, as for the `patronGroups` resource a few lines above. Consider importing `MAX_RECORDS` from `../constants`. 

Additionally, consider a `sortby` clause.